apps/web/components/HomePage/UserEntity.tsx

```diff
@@ -37,7 +37,7 @@ export default function UserEntity(props: {
       target_id: UserState.id,
     };
 
-    fetch(`${process.env.NEXT_PUBLIC_BACKEND_BASE_URL}/friendships/valide`, {
+    fetch(`/api/friendships/validate`, {
```

Comment:
This could stay inside an environment variable (`http://localhost/api`)

Reply:
The address should be relative, not absolute, to work in any context, especially without a proper domain name registered on a DNS provider, otherwise you won't understand why it worked on your machine. So http://localhost should definitely not be included.
About the env variable, I plainly dislike them and barely stand them for docker-compose, however they are necessary in the context. The less you have, the better, they lead to too many security leaks. Best practice would be to use docker-swarm and secrets, however the subject prevents us from doing so